llvm/lib/Transforms/Yk/ShadowStack.cpp

```diff
@@ -216,7 +216,7 @@ class YkShadowStack : public ModulePass {
           // whatever order we encounter them, but we may be able to waste less
           // space (to padding) by sorting them by size.
           size_t Align = AI->getAlign().value();
-          SFrameSize = ((SFrameSize + (Align - 1)) / Align) * Align;
```

Comment:
So it wasn't wrapping as we wanted?

Reply:
We just forgot the cast to int, so `/ Align * Align` cancelled each other out.